### PR TITLE
Removed Uploading of perf result to git repo 'zenith-perf-data'

### PR DESCRIPTION
**Pull request by abhigets**
## Problem
We were archiving the pref benchmarks to 

- neon DB
- git repo `zenith-perf-data`

As the pref batch ran in parallel when the uploading of results to zenith-perf-data` git repo resulted in merge conflicts.
Which made the run flaky and as a side effect the build started failing .

The problem is been expressed in https://github.com/neondatabase/neon/issues/5160 

## Summary of changes
As the results were not used from the git repo it was redundant hence in this PR cleaning up the results uploading of of perf results to git repo
The shell script `generate_and_push_perf_report.sh` was using a py script [git-upload](https://github.com/neondatabase/neon/compare/remove-perf-benchmark-git-upload?expand=1#diff-c6d938e7f060e487367d9dc8055245c82b51a73c1f97956111a495a8a86e9a33) and [scripts/generate_perf_report_page.py](https://github.com/neondatabase/neon/pull/6590/files#diff-81af2147e72d07e4cf8ee4395632596d805d6168ba75c71cab58db2659956ef8) which are not used anywhere else in repo hence also cleaning that up 

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [ ] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

## Checklist before merging

- [ ] Do not forget to reformat the commit message to not include the above checklist

### CODE REVIEW

File: scripts/generate_and_push_perf_report.sh [hunk 1]
```diff
@@ -8,17 +8,3 @@ SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
...
-. "$(poetry env info --path)"/bin/activate
-
-echo "Uploading perf result to zenith-perf-data"
-scripts/git-upload \
-    --repo=https://"$VIP_VAP_ACCESS_TOKEN"@github.com/neondatabase/zenith-perf-data.git \
-    --message="add performance test result for $GITHUB_SHA neon revision" \
-    --branch=master \
-    copy "$REPORT_FROM" "data/$REPORT_TO" `# COPY FROM TO_RELATIVE`\
-    --merge \
-    --run-cmd "python $SCRIPT_DIR/generate_perf_report_page.py --input-dir data/$REPORT_TO --out reports/$REPORT_TO.html"
```
Comment (bayandin) [inline scripts/generate_and_push_perf_report.sh:24]: It seems we can delete `generate_perf_report_page.py` as well.

**Review by bayandin — COMMENTED** [review]
You need to add `run-benchmarks` label to the PR to get benchmarks triggered

File: scripts/generate_and_push_perf_report.sh (hunk 1 above)
Comment (abhigets) [reply scripts/generate_and_push_perf_report.sh:24]: yep generate_perf_report_page.py not used anywhere else

Comment (abhigets) [reply scripts/generate_and_push_perf_report.sh:24]: Done

**Comment by abhigets** [comment]
> You need to add `run-benchmarks` label to the PR to get benchmarks triggered

label added to PR

**Review by bayandin — APPROVED** [review]

**Pull request merged**